agent-bootstrap/src/main/java/io/opentelemetry/auto/config/Config.java

```diff
@@ -638,4 +595,31 @@ public static Config get(final Properties properties) {
       return new Config(properties, INSTANCE);
     }
   }
+
+  public boolean isIntegrationEnabled(
+      final SortedSet<String> integrationNames, final boolean defaultEnabled) {
+    return integrationEnabled(integrationNames, defaultEnabled);
+  }
+
+  private Set<Integer> getIntegerRangeSettingFromEnvironment(
+      final String name, final Set<Integer> defaultValue) {
+    final String value = getSettingFromEnvironment(name, null);
+    try {
+      return value == null ? defaultValue : parseIntegerRangeSet(value, name);
+    } catch (final NumberFormatException e) {
+      log.warn("Invalid configuration for " + name, e);
+      return defaultValue;
+    }
+  }
+
+  private Set<Integer> getPropertyIntegerRangeValue(
+      final Properties properties, final String name, final Set<Integer> defaultValue) {
+    final String value = properties.getProperty(name);
+    try {
+      return value == null ? defaultValue : parseIntegerRangeSet(value, name);
+    } catch (final NumberFormatException e) {
+      log.warn("Invalid configuration for " + name, e);
+      return defaultValue;
+    }
+  }
```

Comment:
can you reduce moving things around in this PR? it will help with merge conflicts as i'm still merging from https://github.com/DataDog/dd-trace-java